snmp/datadog_checks/snmp/data/default_profiles/fortinet-fortigate.yaml

```diff
@@ -185,6 +185,31 @@ metrics:
           OID: 1.3.6.1.4.1.12356.101.3.2.1.1.2
           name: fgVdEntName
 
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      OID: 1.3.6.1.4.1.12356.101.3.2.1
+      name: fgVdTable
+    symbols:
+      - name: fgVd
```

Comment:
wondering if we should rename this to something more understandable like `fgVirtualDomain`